src/hooks/pool/useGhoPoolFormattedReserve.ts

```diff
@@ -0,0 +1,40 @@
+import { ReservesDataHumanized } from '@aave/contract-helpers';
+import { formatGhoReserveData, GhoReserveData } from '@aave/math-utils';
+import memoize from 'micro-memoize';
+import { MarketDataType } from 'src/ui-config/marketsConfig';
+import { GHO_SYMBOL } from 'src/utils/ghoUtilities';
+
+import { useGhoPoolsReserve } from './useGhoPoolReserve';
+import { usePoolsReservesHumanized } from './usePoolReserves';
+import { combineQueries } from './utils';
+
+const selector = memoize((ghoReserveData: GhoReserveData, reservesData: ReservesDataHumanized) => {
```

Comment:
I think the memoize here might not be necessary. I don't think there's anything computationally expensive here that we couldn't just re-run anytime the hook was called.